tests/foreman/cli/test_repository.py

```diff
@@ -2446,18 +2445,17 @@ def test_positive_local_directory_sync(self):
 
         """
         # Making Setup For Creating Local Directory using Pulp Manifest
-        ssh.command("yum -y install python-pulp-manifest")
-        ssh.command("mkdir {}".format(CUSTOM_LOCAL_FOLDER))
-        ssh.command("touch {0} && pulp-manifest {1}".
-                    format(CUSTOM_LOCAL_FILE, CUSTOM_LOCAL_FOLDER))
+        ssh.command("mkdir -p {}".format(CUSTOM_LOCAL_FOLDER))
```

Comment:
perhaps removing that comment saying:
# Making Setup For Creating Local Directory using Pulp Manifest